app/boards/arm/kbdfans_tofu65/kbdfans_tofu65_v2.dts

```diff
@@ -115,3 +117,6 @@ zephyr_udc0: &usbd {
     status = "okay";
 };
 
+&layout_65_ansi {
+    transform = <&default_transform>;
+};
```

Comment:
Not necessary for you to do this right now, but for anyone finding this later, would be nice to add support for the other layouts found at https://kbdfans.com/products/tofu60-2-0?variant=41382653886603 e.g.:

![image](https://github.com/user-attachments/assets/5a91935c-66d5-41c3-a110-0a4431dee430)

Reply:
That's for the tofu60, the tofu65 has only one layout https://kbdfans.com/products/tofu65-2-0?_pos=3&_sid=bb6d96f98&_ss=r

Reply:
Oh indeed. Thanks